src/components/SendAsset/SendAsset.js

```diff
@@ -156,9 +163,12 @@ const SendAsset = ({
     if (!feeInfo || !selectedContact || !assetData) {
       // something went wrong
       Toast.show({
-        message: t('toast.cannotSendAsset'),
-        emoji: 'woman-shrugging',
-        supportLink: true,
+        message: t('toast.transactionReverted'),
+        emoji: 'hushed',
+        autoClose: false,
+        onPress: () => {
+          openUrl(transactionRevertedAricleURL);
+        },
       });
```

Comment:
Seems like this code is repeated couple of times, and each call site needs to get firebase remote config settting, etc. Could we move it a common function in utils? e.g. `showTransactionRevertedToast` in `utils/transactions`?

Reply:
Done.